.github/workflows/test_develop.yml

```diff
@@ -14,7 +14,7 @@ jobs:
   trigger_tests:
     strategy:
       matrix:
-        branch: [develop, release/cycle-3.120, release/cycle-3.123, release/cycle-3.124]  # List other branches here
+        branch: [develop, release/cycle-3.120, release/cycle-3.123, release/cycle-3.124, release/cycle-3.125]  # List other branches here
```

Comment:
issue: hmm weird this shows up here, maybe because it was develop before

Reply:
💥